### PR TITLE
Bug/supplied peers not being read from json

### PR DESCRIPTION
**Pull request by engineerkatie**

### CODE REVIEW

File: libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp [hunk 1]
```diff
@@ -223,6 +240,11 @@ void MuddleLearnerNetworkerImpl::PushUpdateBytes(UpdateType const &type_name, By
     auto next_ones = alg_->GetNextOutputs();
 
     Peers peers;
+    if (next_ones.empty())
+    {
+      FETCH_LOG_WARN(LOGGING_NAME, "PushUpdateBytes(2) got no peers from alg to send to.");
```
Comment (JuanBesa) [inline libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp]: What is (2)?

Shouldn't it be "... **has** no peers...?

File: libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp [hunk 2]
```diff
@@ -74,13 +74,14 @@ void MuddleLearnerNetworkerImpl::Setup(MuddlePtr mud, StorePtr update_store)
...
     byte_array::ConstByteArray bytes;
     double                     proportion;
     double                     random_factor;
 
     auto source = std::string(fetch::byte_array::ToBase64(from));
 
-    buf >> type_name >> bytes >> proportion >> random_factor;
+    buf >> algo_name >> type_name >> bytes >> proportion >> random_factor;
 
     std::cout << "from:" << source << ", "
```
Comment (JuanBesa) [inline libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp:86]: Would it be useful to add algo_name and type_name to the cout message?

**Review by JuanBesa — DISMISSED** [review]
Please see comments

File: libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp (hunk 2 above)
Comment (engineerkatie) [reply libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp:86]: Yes, I should do that.

File: libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp (hunk 1 above)
Comment (engineerkatie) [reply libs/dmlf/src/colearn/muddle_learner_networker_impl.cpp]: (2) is the second message of that form. "All error messages should be unique in the source"

**Comment by engineerkatie** [comment]
Doing this in the private repo instead.

**Pull request closed without merging**